src/v2/Apps/Artwork/Components/ArtworkSidebar/CreateArtworkAlertSection.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { Separator, Spacer, Flex, Button, BellIcon, Text } from "@artsy/palette"
+
+export const CreateArtworkAlertSection: React.FC = () => {
+  return (
+    <>
+      <Spacer m={2} />
+      <Separator />
+      <Spacer m={2} />
+      <Flex
+        flexDirection="row"
+        py={1}
+        alignItems="center"
+        justifyContent="space-between"
+      >
+        <Flex flex={1} justifyContent="flex-start">
+          <Text variant="xs">
+            Be notified when a similar piece is available
+          </Text>
+        </Flex>
+        <Flex flex={1} justifyContent="flex-end">
+          <Button variant="secondaryOutline" size="small">
+            <BellIcon mr={0.5} fill="currentColor" />
+            <Text variant="xs">Create Alert</Text>
+          </Button>
+        </Flex>
+      </Flex>
+      <Spacer m={2} />
+    </>
```

Comment:
**suggestion**: small updates 😅

```tsx
<Box my={2}>
  <Separator mb={2} />

  <Flex
    flexDirection="row"
    py={1}
    alignItems="center"
    justifyContent="space-between"
  >
    <Text variant="xs" mr={2}>
      Be notified when a similar piece is available
    </Text>
    <Button variant="secondaryOutline" size="small">
      <BellIcon mr={0.5} fill="currentColor" />
      <Text variant="xs">Create Alert</Text>
    </Button>
  </Flex>
</Box>
```

Reply:
Nice thanks!  Box seems like the right thing to do here 🤩.

For the removal of the `<Flex>` containers though I think I disagree because in mobile view the text is too close to the button and looks weird 🤔 

| Without Flex Container | With Flex Container |
|--|--|
|<img width="415" alt="Screenshot 2022-03-10 at 14 44 42" src="https://user-images.githubusercontent.com/21178754/157674530-0ab77ae3-21ff-4953-a953-988b0a5494df.png">|<img width="419" alt="Screenshot 2022-03-10 at 14 43 51" src="https://user-images.githubusercontent.com/21178754/157674535-02baba35-756d-4fe7-9a0f-b62fa3dc2320.png">|

Reply:
Can you please tell at what resolution such a problem is observed? I have tested and I have not reproduced it

https://user-images.githubusercontent.com/3513494/157676166-c3ed6454-29c1-41af-8b7c-fb1029527944.mp4

Reply:
oh yeah, of course this behavior exists in devices like iphone 6/7/8
<img width="557" alt="Screenshot 2022-03-10 at 14 56 03" src="https://user-images.githubusercontent.com/21178754/157676742-bc564a97-601e-45f3-9d6b-0f9599b5fd78.png">

Reply:
<img width="565" alt="Screenshot 2022-03-10 at 14 59 24" src="https://user-images.githubusercontent.com/21178754/157677288-8f4e97f4-85c9-464b-a170-25242476104f.png">

Reply:
Nice. thanks for checking!